src/data/resolvers/user.ts

```diff
@@ -140,10 +140,16 @@ const getUserLock = async (
     ClientType.TokenLockingClient,
     colonyAddress,
   );
-  const votingReputationClient = await colonyManager.getClient(
-    ClientType.VotingReputationClient,
-    colonyAddress,
-  );
+  let votingReputationClient;
+
+  try {
+    votingReputationClient = await colonyManager.getClient(
+      ClientType.VotingReputationClient,
+      colonyAddress,
+    );
+  } catch (error) {
+    console.error(error);
```

Comment:
Removing the console.log should do the trick to remove the error. Otherwise, this PR fixes the main error.